### PR TITLE
Changed Modal window height

### PR DESCRIPTION
**Pull request by prahaladhchandrahasan**
Changed Modal window height as per request. Addition to the closed PR: https://github.com/hawtio/hawtio-integration/pull/145

### CODE REVIEW

**Review by tadayosi — CHANGES_REQUESTED** [review]

File: plugins/runtime/threads/thread-dump-modal.component.ts [hunk 1]
```diff
@@ -21,7 +21,11 @@ namespace Runtime {
         <h4 class="modal-title">Thread Dump</h4>
       </div>
       <div class="modal-body">
-        <pre>{{$ctrl.dumpedThreads}}</pre>
+        <div class="row">
+          <div class="col-md-12">
+            <pre style="height:80vh">{{$ctrl.dumpedThreads}}</pre>
```
[the remark this answers is not in the document]
Comment (prahaladhchandrahasan) [reply plugins/runtime/threads/thread-dump-modal.component.ts:26]: okay will resolve this

File: plugins/runtime/threads/thread-dump-modal.component.ts [hunk 2]
```diff
@@ -21,7 +21,11 @@ namespace Runtime {
         <h4 class="modal-title">Thread Dump</h4>
       </div>
       <div class="modal-body">
-        <pre>{{$ctrl.dumpedThreads}}</pre>
+        <div class="row">
+          <div class="col-md-12">
+              <pre class = "thread-dump-modal-output" >{{$ctrl.dumpedThreads}}</pre>
```
Comment (tadayosi) [inline plugins/runtime/threads/thread-dump-modal.component.ts:26]: Format here better pls.
```
<pre class="thread-dump-modal-output">
```

File: plugins/runtime/threads/thread-dump-modal-.component.less [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.thread-dump-modal-output{
```
Comment (tadayosi) [inline plugins/runtime/threads/thread-dump-modal-.component.less:1]: you don't need to create new one. just put it in the existing `.less` file.

File: plugins/runtime/threads/thread-dump-modal-.component.less [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+  height:80vh;
```
Comment (tadayosi) [inline plugins/runtime/threads/thread-dump-modal-.component.less:2]: also, don't forget to add space ' ' between name and value. > `height: 80vh;`
take care of looking more.

**Review by tadayosi — CHANGES_REQUESTED** [review]

File: plugins/runtime/threads/thread-dump-modal.component.ts [hunk 5]
```diff
@@ -21,7 +21,11 @@ namespace Runtime {
         <h4 class="modal-title">Thread Dump</h4>
       </div>
       <div class="modal-body">
-        <pre>{{$ctrl.dumpedThreads}}</pre>
+        <div class="row">
+          <div class="col-md-12">
+              <pre class = "thread-dump-modal-output">{{$ctrl.dumpedThreads}}</pre>
```
Comment (tadayosi) [inline plugins/runtime/threads/thread-dump-modal.component.ts]: formatting issue not yet resolved.

**Review by tadayosi — CHANGES_REQUESTED** [review]

File: plugins/runtime/threads/thread-dump-modal.component.ts (hunk 5 above)
Comment (prahaladhchandrahasan) [reply plugins/runtime/threads/thread-dump-modal.component.ts]: please check the next commit

File: plugins/runtime/threads/threads.component.less [hunk 6]
```diff
@@ -20,3 +20,6 @@
     }
   }
 }
+.thread-dump-modal-output{
```
[the remark this answers is not in the document]
Comment (prahaladhchandrahasan) [reply plugins/runtime/threads/threads.component.less]: done

**Review by tadayosi — APPROVED** [review]
looks good, finally. thanks!

**Pull request merged**